src/lager_transform.erl

```diff
@@ -200,15 +200,15 @@ do_transform(Line, SinkName, Severity, Arguments0, Safety) ->
                   [],
                   %% trick the linter into avoiding a 'term constructed but not used' error:
                   %% (fun() -> {error, lager_not_running} end)()
-                  [{call, Line, {'fun', Line, {clauses, [{clause, Line, [],[], [{tuple, Line, [{atom, Line, error},{atom, Line, lager_not_running}]}]}]}}, []}]
+                  [{call, Line, {'fun', Line, {clauses, [{clause, Line, [],[], [{tuple, erl_anno:set_generated(true, Line), [{atom, Line, error},{atom, Line, lager_not_running}]}]}]}}, []}]
           },
           %% {undefined, _, _} -> {error, {sink_not_configured, Sink}};
           {clause,Line,
                   [{tuple,Line,
                           [{atom,Line,undefined},{var,Line,'_'},{var,Line,'_'}]}],
                   [],
                   %% same trick as above to avoid linter error
-                  [{call, Line, {'fun', Line, {clauses, [{clause, Line, [],[], [{tuple,Line, [{atom,Line,error}, {tuple,Line,[{atom,Line,sink_not_configured},{atom,Line,SinkName}]}]}]}]}}, []}] 
+                  [{call, Line, {'fun', Line, {clauses, [{clause, Line, [],[], [{tuple, erl_anno:set_generated(true, Line), [{atom,Line,error}, {tuple,Line,[{atom,Line,sink_not_configured},{atom,Line,SinkName}]}]}]}]}}, []}]
```

Comment:
Same here. No `fun` needed.

Reply:
Sure. Thanks for checking this. I went the lazy route, it seems, but glad you had a change to review.